hasher-matcher-actioner/hmalib_extensions/action_performer_example.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import typing as t
+
+
+from hmalib.common.messages.action import ActionMessage
+from hmalib.common.extensions.action_performers import ActionPerformerExtensionBase
+
+
+class ActionPerformerExtensionExample(ActionPerformerExtensionBase):
+    """
+    Example implementation of an actioner extension
+    This example expects one of the keyword args to be "defined_keyword_arg"
+    It prints that argument along with the other optional kwargs and the
+    additional_fields on the action message.
+    """
+
+    @classmethod
+    def perform_action_impl(
+        cls, message: ActionMessage, additional_kwargs: t.Dict[str, str]
```

Comment:
Would love if we could actually use kwarg names in additional_kwargs instead of a dict.